web/packages/shared/components/AccessRequests/NewRequest/RequestCheckout/RequestCheckout.tsx

```diff
@@ -417,25 +518,59 @@ export function RequestCheckout<T extends PendingListItem>({
                     appsGrantedByUserGroup.length > 0 && (
                       <AppsGrantedAccess apps={appsGrantedByUserGroup} />
                     )}
-                  {isResourceRequest && (
+                  <Flex flexDirection="column" gap={2} mt={4}>
+                    <Text bold>Request type</Text>
+                    <HoverTooltip
+                      tipContent={longTermDisabledReason}
+                      placement="left"
+                    >
+                      <Toggle
+                        isToggled={isLongTerm}
+                        onToggle={() =>
+                          setRequestKind(
+                            isLongTerm
+                              ? RequestKind.ShortTerm
+                              : RequestKind.LongTerm
+                          )
+                        }
+                        disabled={longTermDisabled}
+                        css={
+                          longTermDisabled && `cursor: not-allowed !important;`
```

Comment:
Nit: is there any reason to differ from the regular styling for a disabled toggle?

Reply:
Bump, I'd stick to the regular styling if we can.